src/components/views/donate/OnTime/SelectTokenModal/QRCodeDonation/QRDonationCard.tsx

```diff
@@ -166,9 +169,13 @@ export const QRDonationCard: FC<QRDonationCardProps> = ({
 				? parsedLocalStorageItem[projectAddress?.address!]
 				: null;
 
-			const retDraftDonation = !!draftDonationId
-				? await retrieveDraftDonation(Number(draftDonationId))
-				: null;
+			let retDraftDonation;
+
+			if (!!draftDonationId) {
```

Comment:
@Meriem-BM 
You just used `!!draftDonationId` for casting it to `boolean` ?
Isn't better to do this instead of that tricky way? `Boolean(draftDonationId)`

Reply:
Yeah, that also would work